.github/ISSUE_TEMPLATE/pre-submission.md

```diff
@@ -0,0 +1,145 @@
+title: Pre-submission Inquiry for Ontology [NAME]
+labels: [ pre-submission ]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Do _NOT_ use this form to register a new ontology with the OBO Foundry. Please read the instructions provided here: http://obofoundry.org/docs/NewOntologyRegistrationInstructions.html.
+  - type: input
+    id: title
+    attributes:
+      label: Title
+      description: What is the full title of your ontology? Please do not include redundant acronyms.
+      placeholder: ex. Chemical Entities of Biological Interest
+    validations:
+      required: true
+  - type: input
+    id: slug
+    attributes:
+      label: Short Description
+      description: Please provide a short description of your ontology in one sentence.
+    validations:
+      required: true
+  - type: input
+    id: idspace
+    attributes:
+      label: Identifier Space
+      description: What is the prefix for the identifier space associated with your ontology? This is typically all capital letters. This *must* not conflict with any entries in the [Bioregistry](https://bioregistry.io/registry) nor [BioPortal](https://bioportal.bioontology.org). See OBO Principle [FP-003 "URI/Identifier Space"](https://obofoundry.org/principles/fp-003-uris.html).
+      placeholder: ex. GO
+    validations:
+      required: true
+  - type: dropdown
+    id: domain
+    validations:
+      required: true
+    attributes:
+      label: Domain
+      description: Please choose the domain that best describes your ontology
+      options:
+        - upper
+        - anatomy and development
+        - health
+        - phenotype
+        - investigations
+        - environment
+        - biochemistry
+        - microbiology
+        - agriculture
+        - diet, metabolomics, and nutrition
+        - biological systems
+        - information technology
+        - organisms
+        - information
+        - simulation
+  - type: markdown
+    attributes:
+      value: |
+        # Links
+
+        This section of the ontology request form contains links to various resources associated with your ontology.
+  - type: input
+    id: repository
+    attributes:
+      label: Source Code Repository
+      description: What is the URL of the repository? Typically, this will be a link to GitHub.
+      placeholder: e.g. https://github.com/obophenotype/uberon/
+    validations:
+      required: true
+  - type: input
+    id: tracker
+    attributes:
+      label: Issue Tracker
+      description: What is the URL of the public issue tracker? Typically, this will be the link to the GitHub repository with a trailing `/issues` URL part.
+      placeholder: e.g. https://github.com/obophenotype/uberon/issues
+    validations:
+      required: true
+  - type: input
```

Comment:
to make this more useful, also include a dropdown that asks in what format the ontology edit file is (OBO flat format, OWL/XML, OFN, etc.)

Reply:
I think the form should be short, because we wont really look at an ontology submitted this way.. Why do you think the format would be necessary for a presubmission inquiry?

Reply:
In light of https://github.com/OBOFoundry/OBOFoundry.github.io/pull/2285#pullrequestreview-1287722766, I think i missed the point. It would be helpful in the future to add to the SOPs for the operations committee that when you make a PR, you should include context and not just a reference saying "we talked about this in the meeting". This way, people who didn't attend the meetings can understand *why* a PR is made

Reply:
You are right - for PRs. In my world view it is not a PR if it is in draft state! Draft means also, that the PR motivation text is still in Draft mode.. Once it turns green, there should be a good text with context though, cc @erik-whiting